test/README.md

```diff
@@ -90,3 +90,13 @@ The program will wait until you launch the Debug mode from VS Code (next step).
 Launch debug session by selecting Run/Start Debugging from top menu.
 
 Further information [here](https://github.com/Microsoft/vscode-go/wiki/Debugging-Go-code-using-VS-Code).
+
+## Miscellaneous
+
+### Prune namespaces
+
+In case you stopped a test execution, the test framework won't delete the namespaces in the cluster. So to avoid waste of resources in the cluster, you need to manually delete these namespaces. In order to ease this, now we have a namespace history log and an utility to do this by running:
+
+```sh
+go run utils/prune_namespaces.go
```

Comment:
@radtriste I provided this function to be used by the pipeline and I think it could be useful for us too (I tend to stop test executions when running the tests locally if I spot something or don't want to wait for the test to finish).

Reply:
cool thing ! will have a look.
maybe we could also add a section in jenkins pipeline(s) to always call this script after the tests ran, so it can clean if any interruption :)

Reply:
and could the `run-tests.sh` script call this one after running the tests ?
(if no `keep-namespace` is set of course)
that would give a second security I think

Reply:
It makes sense to prune namespaces in run-tests.sh (in case when we stop the test execution) and in the pipeline (in case when the pipeline stops).
Added.